README.md

```diff
@@ -4,33 +4,39 @@
 * [API Documentation](https://docs.teaching-vacancies.service.gov.uk) (External link)
 * [API Keys](#api-keys)
 * [Dependencies](#dependencies)
+* [GovUK PaaS](/documentation/govuk-paas.md)
 * [Importing school data](#importing-school-data)
+* [Logging](/documentation/logging.md)
 * [Misc](#misc)
 * [Running the tests](#running-the-tests)
+* [Secrets](#secrets)
+* [Setup](#setup) (Start here!)
 * [Troubleshooting](#troubleshooting)
 * [User accounts](#user-accounts)
 
 ## User accounts 
 
-Before you can log in to the application locally you will need a __DfE Sign-in__ and an __invitation to join Teaching
-Vacancies__. Talk to the team to get these set up.
+Before you can log in to the application locally, you will need a __DfE Sign-In__ and an __invitation to join Teaching
```

Comment:
You can create a DSI account on the sign-in page. I think it's the first step before getting invited?